examples/Reading in linearised models.ipynb

```diff
@@ -0,0 +1,376 @@
+{
```

Comment:
<p>Can you pass shock sizes to silence those warnings? Even if you just use the default. Since they're all the same you can just pass a covariance matrix:</p><p><code>mod1.compute_stationary_covariance_matrix(shock_cov_matrix=np.eye(3) * 0.01)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jessegrabowski/gEconpy/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>